cmd/init/init.go

```diff
@@ -173,6 +180,35 @@ func getDeployment(namespace string) (*appsv1.Deployment, string, error) {
 	return d, container, nil
 }
 
+func supportsPersistentVolumes(namespace string) bool {
+	c, _, currentNamespace, err := k8Client.GetLocal()
+	if err != nil {
+		log.Debugf("deactivating persistent volumes: %s", err.Error())
+		return false
+	}
+	if namespace == "" {
+		namespace = currentNamespace
+	}
+
+	ns, err := namespaces.Get(namespace, c)
+	if err != nil {
+		log.Debugf("deactivating persistent volumes: %s", err.Error())
+		return false
+	}
+
+	if namespaces.IsOktetoNamespace(ns) {
+		return true
+	}
+
+	stClassList, err := c.StorageV1().StorageClasses().List(metav1.ListOptions{})
+	if err != nil {
+		log.Debugf("deactivating persistent volumes: %s", err.Error())
+		return false
+	}
+
+	return len(stClassList.Items) > 0
```

Comment:
we should also check that there's a default storage class, I think some of the issues come from there. 

Also, what about incompatible storage classes? (e.g. the issue we saw with minikube and the volumes not persisting a restart). Maybe we can identify those by name and log a warning?